### ISSUE TITLE
Kingdom favor not updating outside the island

### ISSUE THREAD

**Issue by Kriicz** [runelite/runelite#1748]
favor % doesn't change unless you visit the island

**Comment by sethtroll** [comment]
Yeah, unfortunately, you are unable the grab the updated information until you visit the island. #1267 was just recently merged which will go live on Thursday's update that will make it only display when your on the island.

**Comment by TheArbalist** [comment]
@Adam- I think you may have referenced my feature request by mistake.

**Comment by Adam-** [comment]
Ah yes, sorry

**Comment by TheArbalist** [comment]
No worries. Get some sleep later. :P
